lib/wds-components/app/components/wds-global-navigation.js

```diff
@@ -2,10 +2,12 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
 	classNames: 'wds-global-navigation',
-	classNameBindings: ['searchIsActive:wds-search-is-active'],
+	classNameBindings: ['searchIsActive:wds-search-is-active', 'searchIsAlwaysVisible:wds-search-is-always-visible'],
 
 	searchIsActive: false,
 
+	searchIsAlwaysVisible: Ember.computed.empty('model.fandom_overview'),
```

Comment:
Overaddressed in https://github.com/Wikia/design-system/pull/96/commits/b846f07bdd9c4d44adb0a873d9185cb9228f19bd